### PR TITLE
RPRBLND-2014: Some implemented nodes are not available in RPR addon

### PR DESCRIPTION
**Pull request by VascoPi**
### PURPOSE
Some implemented nodes are not available in RPR addon.

### EFFECT OF CHANGE
Items are sorted now
New nodes are available in UI:
Anisotropic Bsdf
Diffuse Bsdf
Glass Bsdf
Glossy Bsdf
Refraction Bsdf
Translucent Bsdf
Transparent Bsdf
Velvet Bsdf
SubsurfaceScattering
Frame
RPR ImageTexture
RPR BumpMap
RPR NormalMap

### TECHNICAL STEPS
Added sorded_items func.
Registered category items:
ShaderNodeBsdfAnisotropic
ShaderNodeBsdfDiffuse
ShaderNodeBsdfGlass
ShaderNodeBsdfGlossy
ShaderNodeBsdfRefraction
ShaderNodeBsdfTranslucent
ShaderNodeBsdfTransparent
ShaderNodeBsdfVelvet
ShaderNodeSubsurfaceScattering
NodeFrame
RPRShaderNodeImageTexture
RPRShaderNodeBumpMap
RPRShaderNodeNormalMap

### CODE REVIEW

File: src/rprblender/nodes/__init__.py [hunk 1]
```diff
@@ -47,11 +50,12 @@ def poll(cls, context):
         NodeItem('ShaderNodeVolumeInfo'),
         NodeItem('RPRShaderProceduralUVNode'),
         NodeItem('RPRShaderNodeLookup'),
-    ],),
-    RPR_ShaderNodeCategory('RPR_OUTPUT', "Output", items=[
+        NodeItem('ShaderNodeLightFalloff'),
```
Comment (bsavery) [inline src/rprblender/nodes/__init__.py:53]: We don't support this node.  Why add it?

File: src/rprblender/nodes/__init__.py [hunk 2]
```diff
@@ -64,31 +68,46 @@ def poll(cls, context):
...
+        NodeItem('ShaderNodeBsdfGlossy'),
+        NodeItem('ShaderNodeBsdfRefraction'),
+        NodeItem('ShaderNodeBsdfTranslucent'),
+        NodeItem('ShaderNodeBsdfTransparent'),
+        NodeItem('ShaderNodeBsdfVelvet'),
+        NodeItem('ShaderNodeSubsurfaceScattering'),
+        NodeItem('RPRShaderNodeDiffuse'),
+        NodeItem('RPRShaderNodeEmissive'),
+        NodeItem('RPRShaderNodeBlend'),
+    ])),
```
Comment (bsavery) [inline src/rprblender/nodes/__init__.py:83]: So now I'm remembering why these are not included.  We wanted to suggest that people use Principled and Uber with the RPR plugin.  If they have node networks already made they still work.

Comment (VascoPi) [reply src/rprblender/nodes/__init__.py:83]: These will be removed:
RPRShaderNodeDiffuse
RPRShaderNodeEmissive
RPRShaderNodeBlend

How about these nodes?
RPRShaderNodeImageTexture
RPRShaderNodeBumpMap
RPRShaderNodeNormalMap

File: src/rprblender/nodes/__init__.py (hunk 1 above)
Comment (VascoPi) [reply src/rprblender/nodes/__init__.py:53]: Yes, you are right. I'll fix it.

File: src/rprblender/nodes/__init__.py (hunk 2 above)
Comment (bsavery) [reply src/rprblender/nodes/__init__.py:79]: The reason the RPR ones are not there is because they're redundant to the blender ones

Comment (VascoPi) [reply src/rprblender/nodes/__init__.py:79]: Removed.
RPRShaderNodeImageTexture
RPRShaderNodeBumpMap
RPRShaderNodeNormalMap

**Review by DagerD — APPROVED** [review]
Tested. Works ok.

**Review by bnagirniak — APPROVED** [review]
Looks good

**Pull request merged**